line_plot_zoom.js

```diff
@@ -269,9 +269,12 @@ function genomeLineChart() {
 
     // update the PROTEIN structure
     circleData.protein_chain.forEach(function(chain){
-      selectSiteOnProtein(":" + chain + " and " +
-        circleData.protein_site,
-        color_key[circleData.site]);
+      if(!missingData.includes(chain) ||
```

Comment:
Do we want this test to be an `&&` instead of an `||`? If the chain is missing and the protein site is not, won't we have the same problem of the whole protein being selected because the chain is NaN? Or does this only work if the chain is an empty string but not a NaN?

Reply:
Oh! you are absolutely correct! let me push that change.